Configure

```diff
@@ -565,6 +564,7 @@ my %deprecated_disablables = (
     "ripemd" => "rmd160",
     "ui" => "ui-console",
     "heartbeats" => undef,
+    "atexit" => undef,
```

Comment:
I wonder if this actually should not be deprecated and that this would initialize `ossl_no_cleanup` to 1 instead of 0.

Reply:
If I understand things right this proposal is almost equivalent to dropping a call to clean up at process exit completely. It brings us one step closer to solution where we just drop implicit call to clean up  at process exit completely, which I think we should consider.

Reply:
@Sashan could you please reconsider this? I am not asking to make the OPENSSL_NO_ATEXIT more powerfull, I am just asking to keep the _just drop implicit call to clean up_ available by using `no-atexit`.